WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -306,6 +306,11 @@
         <%= f.input :extra_registration_requirements, input_html: { class: disable_form ? "" : "markdown-editor" } %>
 
         <hr>
+        <% if @competition.uses_qualification? %>
+          <%= f.input :qualification_results_reason %>
```

Comment:
Sorry I realised this only now but you're adding a field about `qualification_results_reason` which seems unrelated (and is also included further down in the same form!)

Perhaps a mistake from merging, please fix.

Reply:
Added to the edit below.